src/components/FeaturedCategories.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import low from "../Images/category-low.png"
+import high from "../Images/category-high.png"
+import jordan from "../Images/air-jordan.jpg"
+import "../styles/FeaturedCategories.css"
+const FeaturedCategories = () => {
+  return (
+    <div className="Categories">
+        <h2 className='content'> Categories to Choose From...</h2>
+        <div className="small-container">
+            <div className="row">
+                <div className="col-3">
+                    <p className='img-tag'>Air Max Low</p>
+                <img src={low}></img>
```

Comment:
suggestion: if possible add alt tag in the inside image.

Reply:
will take care from next time